Lesson4/src/com/home/MilitaryBase.java

```diff
@@ -72,4 +73,11 @@ public int addtoBase(int size) {
     public int getSize() {
         return this.size;
     }
+    public void getRecruitsNameSurname() {
+        System.out.println("Сортированный список служащих:");
+        baseStaff.sort(Comparator.comparing(Person::getSurname));
+        for (int i = 0; i < baseStaff.size(); i++) {
+            System.out.println(baseStaff.get(i).getSurname() + " " + baseStaff.get(i).getName());
```

Comment:
please also print index here, so the list would look like

1. Ivan Ivanov
2. Jack Jackson
3. ...